share/spice/zomato/zomato.css

```diff
@@ -0,0 +1,9 @@
+.zci--zomato .tile--loc .tile__media__img {
+    max-width: 100% !important;
+}
+.zci--zomato .tile--loc .review-count {
```

Comment:
@moollaza I have added this to make the `reviews` count text inlined with `rating`. I observed this on DDH8 but not on DuckPAN. Let me know would this be necessary when the IA is on live website. Thank you.
PFA.
![screen shot 2015-08-17 at 3 23 27 pm](https://cloud.githubusercontent.com/assets/2561485/9302516/c830b158-44f4-11e5-8bb2-2719dbee5e0e.png)